angular-ui/src/app/sidebar/recommendations/recommendations.component.html

```diff
@@ -0,0 +1,6 @@
+
+<!DOCTYPE html>
+<h2>Movie Recommendation</h2>
+<div (click)="startSpin()" style="cursor: pointer;">
```

Comment:
I would recommend making a separate button to swap movie recommendations instead of making the movie-item clickable. Currently the user doesn't know they can click there to see other recommendations. Also, if you click "more details" or the "Watched" or "watch list" buttons, it changes the movie rather than doing the functionality of those buttons.